lib/debug/variable_inspector.rb

```diff
@@ -0,0 +1,138 @@
+# frozen_string_literal: true
+
+require_relative 'limited_pp'
+
+module DEBUGGER__
+  class VariableInspector
+    def indexed_members_of(obj, start:, count:)
+      return [] if start > (obj.length - 1)
+
+      capped_count = [count, obj.length - start].min
```

Comment:
Can you add the comment why we need to compare  `count` and `obj.length - start`? Sorry, I could not understand the reason.

Reply:
It's to keep the iteration from running off the end of the array. This used to give extra `nil`s at the end. See https://github.com/ruby/debug/pull/1001/commits/68374596f78c9d32141427d6aec5462a9d118b88

I can add a comment.

Could you please run the CI?

Reply:
Done.